account-trigger-zip-code.cls

```diff
@@ -1,19 +1,46 @@
 trigger populateSchoolDistrictBasedOnZip on Account (before update, before insert) {
 
+// Create and add zipcodes into a Set. Criteria set for only Churches with zipcodes populated and blank School Districts.
```

Comment:
@sungysang In order to push any code into a production org you need a unit test.

Reply:
@Julian88Tex Oh awesome I actually just went over creating tests with David... will look into it some more